### PR TITLE
my commit

### PR DESCRIPTION
**Pull request by dergachovda**

### CODE REVIEW

File: Dima D/test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi
```
Comment (stanislav-parkhomenko) [inline Dima D/test.txt]: You should have written a Hello World program

File: Dima D/test.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Hello World
```
Comment (parkhomenko) [inline Dima D/test.txt]: I also disagree

Comment (Stanislavk1982) [inline Dima D/test.txt]: Commit 1 from Stas

Comment (dergachovda) [inline Dima D/test.txt]: ok

**Pull request merged**